test/fixtures/usage_charges.json

```diff
@@ -0,0 +1,23 @@
+{
+  "usage_charges": [
+    {
+      "id": 359376005,
+      "description": "1000 emails",
+      "price": "1.00",
+      "recurring_application_charge_id": 654381177,
+      "billing_on": "2015-07-08",
+      "balance_remaining": "82.00",
+      "balance_used": "18.00"
+
+    },
+    {
+      "id": 359376006,
+      "description": "1000 emails",
+      "price": "1.00",
+      "recurring_application_charge_id": 654381177,
```

Comment:
If these are for the same recurring charge they should have the same `balance_remaining`?